UIKit_DZ/UIKit_DZ/AuthScreen/AuthViewController.swift

```diff
@@ -3,5 +3,171 @@
 
 import UIKit
 
-/// Vaka
-final class AuthViewController: UIViewController {}
+/// Экран приложения с авторизацией
+final class AuthViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let authLabel = "Авторизация"
+        static let loginLabel = "Логин"
+        static let passwordLabel = "Пароль"
+        static let loginTextField = "Введите почту"
+        static let passwordTextField = "Введите пароль"
+    }
+
+    // MARK: - Visual Components
+
+    /// лейбл с именем кафе
+    private lazy var nameCafeLabel: UIImageView = {
+        let image = UIImageView()
+        image.image = UIImage(named: "piconLabel")
```

Comment:
```suggestion
        image.image = UIImage(named: "piconLabel")
```
"piconLabel" в константы все строки